From 80140c03aab5dcb699da042c13c0a00d09e2f3ed Mon Sep 17 00:00:00 2001
From: fossdd <fossdd@pwned.life>
Date: Sun, 12 Jan 2025 22:33:33 +0100
Subject: [PATCH] Remove GNOME 47 warning

The referenced MR got merged in gnome-shell master and 47, and the MR is
included in the latest Alpine release. Therefore this warning is not
required.

Ref: https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/4148
Signed-off-by: fossdd <fossdd@pwned.life>
---
 src/mh_application.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mh_application.py b/src/mh_application.py
index 667b0a9..32ac06b 100644
--- a/src/mh_application.py
+++ b/src/mh_application.py
@@ -49,7 +49,7 @@ class MirrorhallApplication(Adw.Application):
 
         settings = Gio.Settings.new("eu.nokun.MirrorHall")
         if settings.get_boolean('first-run') == True:
-            MhErrorDialogs(win).mutter_47_warning()
+            # MhErrorDialogs(win).mutter_47_warning()
             # settings.set_boolean('first-run', False)
 
         self.inhibit(
@@ -318,4 +318,4 @@ def main(version):
 
     debug = os.getenv("MH_MULTI")
     app = MirrorhallApplication(net_service, dbus_service, sink_port, debug)
-    return app.run(sys.argv)
\ No newline at end of file
+    return app.run(sys.argv)
-- 
2.48.0

